addons/barcodes/models/ir_http.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def session_info(self):
+        res = super(IrHttp, self).session_info()
+        res['max_time_between_keys_in_ms'] = int(
+            self.env['ir.config_parameter'].sudo().get_param('barcode.max_time_between_keys_in_ms', default='55'))
```

Comment:
don't you need to parseInt(x, 10)?

Reply:
in the js i mean

Reply:
not sure I understand your comment. there is a `int(` just above

Reply:
didn't saw it, sorry about that :)